listener/src/event_handler/util.rs

```diff
@@ -0,0 +1,20 @@
+use std::sync::Arc;
+use graph::components::store::EntityKey;
+use web3::types::Log;
+
+use crate::SUBGRAPH_ID;
+
+pub fn entity_id_from_log(log: &Arc<Log>) -> String {
```

Comment:
Is the entity id the transaction hash?

Reply:
it's tx_hash appended by the index of the event inside the block (which should be unique)